src/controller/python/chip/yaml/format_converter.py

```diff
@@ -23,8 +23,49 @@
 import binascii
 
 
+def _substitute_in_config_variables(field_value, config_values: dict):
+    ''' Substitutes values that are config variables.
+
+    YAML values can contain a string of a configuration variable name. In these instances we
+    substitute the configuration variable name with the actual value.
+
+    # TODO This should also substitue any saveAs values as well as perform any required
+    # evaluations.
+
+    Args:
+        'field_value': Value as extracted from YAML.
+        'config_values': Dictionary of global configuration variables.
+    Returns:
+        Value with all global configuration variables substituted with the real value.
+    '''
+    if (type(field_value) is dict):
```

Comment:
Suggest `from collections.abc import MutableMapping, MutableSequence`, then

`if instance(field_value, collections.abc.MutableMapping)` --> for dict

`if instance(field_value, collections.abc.MutableSequence)` --> for list

This would scale to derived classes and any user of ABC.